feature_testing.m

```diff
@@ -0,0 +1,87 @@
+%% Kyle's Dumb Tests
```

Comment:
Probably not needed haha

Reply:
I'd be fine with getting rid of this whole file.